test/fixtures/hooks/test_hook.exs

```diff
@@ -0,0 +1,3 @@
+use Kitto.Hooks.DSL
+
+hook :hello, do: broadcast! :hello, %{text: "Hello World"}
```

Comment:
Yeah, I saw that. Unfortunately, I cannot make the `conn` object available in hooks without getting that warning, as not every hook will necessarily use the `conn` object.